### PR TITLE
GEODE-5288: fix execute function on region call and create more tests…

### PR DESCRIPTION
**Pull request by jinmeiliao**
… to improve coverage.

Co-authored-by: Michael Oleske <moleske@pivotal.io>

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java [hunk 1]
```diff
@@ -73,7 +72,7 @@ public ResultModel executeFunction(
       // find the members based on the groups or members
       dsMembers = findMembers(onGroups, onMembers);
     } else {
-      dsMembers = findMembersForRegion(onRegion);
+      dsMembers = findAnyMembersForRegion(onRegion);
```
Comment (pdxrunner) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java]: `--region` option supersedes `--group` and `--member` show there should be check here for the invalid combination.

Comment (jinmeiliao) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java]: this is checked in the interceptor

Comment (pdxrunner) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java]: OK. I missed that in my first review

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java [hunk 2]
```diff
@@ -322,25 +332,39 @@ public static ResultModel createInfo(String message) {
...
   }
 
+
+
   public static ResultModel createMemberStatusResult(List<CliFunctionResult> functionResults,
-      boolean ignoreIfFailed) {
-    return createMemberStatusResult(functionResults, null, null, ignoreIfFailed);
+      boolean ignoreIgnorable, boolean ignorePartialFailure) {
+    return createMemberStatusResult(functionResults, null, null, ignoreIgnorable,
+        ignorePartialFailure);
```
Comment (pdxrunner) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java]: +1 to adding `ignorePartialFailure`, improves the `ResultModel` semantics.

**Pull request merged**